### DISCUSSION TITLE
supabase different templates in edge function

### DISCUSSION THREAD

**Discussion by megacherry** [resend/react-email#1678] [Q&A]
Hi guys,

i use resend and supabase (edge functions) to send mails (e.g. https://supabase.com/docs/guides/functions/examples/auth-send-email-hook-react-email-resend).

does anyone know how to use different templates in an edge function?

i had tried postmarkapp back then and there you can store templates that you then use.

is there such a function with resend?

best regards,
Andy

**Comment by gabrielmfern** [comment]
You can, you just need to write all the React templates down you want, and then use them accordingly.
